WalletWasabi/Nostr/NostrExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Linq;
+using System.Net;
+using System.Net.WebSockets;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using NBitcoin.Secp256k1;
+using NNostr.Client;
+
+namespace WalletWasabi.Nostr;
+
+public record NostrCoordinator(string Description, string Name, Uri Uri, Network Network);
+
+public record GingerNostrCoordinator()
+	: NostrCoordinator(
+		Description: "Ginger Coordinator | FREE Remix, FREE under 0.01 BTC, FREE for Wasabi mixed coins | SAFE COINJOINS - Illicit actors are not allowed to participate!",
+		Name: "Ginger Coordinator",
+		Uri: new Uri("https://api.gingerwallet.io/"),
+		Network: Network.Main);
```

Comment:
Please review the wording here. And also if it is ok to be hardcoded. @luzius1089 @CAnorbo